sdk/src/main/java/com/mapbox/maps/NativeMapImpl.kt

```diff
@@ -374,6 +375,14 @@ internal class NativeMapImpl(private val map: MapInterface) :
     map.queryRenderedFeatures(pixel, options, callback)
   }
 
+  override fun queryRenderedFeatures(
+    geometry: RenderedQueryGeometry,
+    options: RenderedQueryOptions,
+    callback: QueryFeaturesCallback
```

Comment:
nit: 

```suggestion
    callback: QueryFeaturesCallback,
```

Reply:
not using it anywhere else in that file, so let's be consistent